### PR TITLE
Floating post action menu

### PR DESCRIPTION
**Pull request by bkrmendy**
# Try it [here](https://utopia.pizza/p/b039eb40-tabby-wandflower/?branch_name=spike-floating-post-action-menu)

https://screenshot.click/06-35-oynf1-fnfqj.mp4

## Description
This PR implements an on-canvas post-action menu (see the video above for a demo).

Additionally, two under-the-hood changes are added
- The post-action menu session is reset on undo
- `editorStayedTheSame` in `editorDispatchInner` considers whether `postActionInteractionSession` changed (without this, the selector picking from the post-action session don't update)

### CODE REVIEW

**Review by seanparsons — APPROVED** [review]
That's real nice!

File: editor/src/components/inspector/inspector.tsx [hunk 1]
```diff
@@ -88,6 +88,8 @@ import { ConditionalSection } from './sections/layout-section/conditional-sectio
 import { treatElementAsFragmentLike } from '../canvas/canvas-strategies/strategies/fragment-like-helpers'
 import { allSelectedElementsContractSelector } from './editor-contract-section'
 import { FragmentSection } from './sections/layout-section/fragment-section'
+import { useErrorOverlayRecords } from '../../core/shared/runtime-report-logs'
```
Comment (enidemi) [inline editor/src/components/inspector/inspector.tsx:91]: 🧹 unused imports

File: editor/src/components/canvas/controls/select-mode/post-action-menu.tsx [hunk 2]
```diff
@@ -10,191 +21,403 @@ import { useDispatch } from '../../../editor/store/dispatch-context'
...
+            )}
+          </FlexColumn>
+        </div>
+      </CanvasOffsetWrapper>
+    )
+  },
+)
+FloatingPostActionMenu.displayName = 'FloatingPostActionMenu'
+
+export const InspectorPostActionMenu = React.memo(() => {
```
Comment (enidemi) [inline editor/src/components/canvas/controls/select-mode/post-action-menu.tsx:391]: is this still used somewhere?

**Review by enidemi — APPROVED** [review]

File: editor/src/components/canvas/controls/select-mode/post-action-menu.tsx (hunk 2 above)
Comment (bkrmendy) [reply editor/src/components/canvas/controls/select-mode/post-action-menu.tsx:391]: great catch

Comment (bkrmendy) [reply editor/src/components/canvas/controls/select-mode/post-action-menu.tsx]: https://github.com/concrete-utopia/utopia/pull/3895/commits/858e248bf03b47ea33e6ca2cfb6f4982b76ec5d1

File: editor/src/components/inspector/inspector.tsx (hunk 1 above)
Comment (bkrmendy) [reply editor/src/components/inspector/inspector.tsx]: https://github.com/concrete-utopia/utopia/pull/3895/commits/858e248bf03b47ea33e6ca2cfb6f4982b76ec5d1

**Pull request merged**